src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java

```diff
@@ -604,6 +626,7 @@ public boolean configure(
             prependParentProjectKey = formData.getBoolean("prependParentProjectKey");
 
 			disableInprogressNotification = formData.getBoolean("disableInprogressNotification");
+			buildState = formData.getString("buildState");
```

Comment:
Also I think this is a bad architectural design to pass the build state to the notifier constructor. That's not a part of configuration, but a part of runtime call